library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundle.wdl

```diff
@@ -0,0 +1,80 @@
+task BuildCellRangerReference {
+  String ref_fasta
+  String gtf_file
+  String ref_name
+
+  String docker = "quay.io/humancellatlas/secondary-analysis-cellranger"
+  String memory = "416 GB"
+  Int boot_disk_size_gb = 12
+  String disk_space = "250"
+  Int cpu = 64
+
+  command {
+    set -e
+
+    wget ${ref_fasta}
+    ref_fasta_tar_filename=$(echo ${ref_fasta} | sed 's:.*/::')
+    gunzip $ref_fasta_tar_filename
```

Comment:
Few little things:
- I think this isn't a tar archive, so probably want to remove tar from the variable name.
- You can download and decompress in parallel
- I think `basename` is a more straightforward way to get the local name

So something like this:
```bash
ref_fasta_filename="$(basename "${ref_fasta}")"
ref_fasta_filename="${ref_fasta_filename%.gz}"
wget -q -O - "${ref_fasta}"| zcat > "$ref_fasta_filename"
```